sass/pages/_assets.scss

```diff
@@ -18,29 +18,109 @@
         margin-bottom: 52px;
     }
 
-    .card {
-        margin: 0;
-        height: 8rem;
-    }
+    .asset-card {
+        display: grid;
+        grid-template-areas:
+            "banner"
+            "desc"
+            "tags";
+        grid-template-rows: 12rem 5.5rem 2rem;
```

Comment:
**Option B**, for comment: https://github.com/bevyengine/bevy-website/pull/394/files#r924157060

```suggestion
        grid-template-rows: 12rem minmax(5.5rem, 1fr) 2rem;
        height: 100%;
```

- `minmax` to allow the description grow, but keep a minimum height
- `height` for the other cards, so that they grow to match the one with the long description

![image](https://user-images.githubusercontent.com/188612/179692203-4e1c381b-aeab-404e-8f27-9ccdbeea0c8c.png)

Reply:
I'm biased against growing the cards vertically; when we were designing the initial mockups we had a lot of discussion about the amount of text that would be appropriate for cards, and we landed on the ratios and everything that we did based on a relatively little amount of text. 

Descriptions should be short and to the point. Longer exposition about a particular asset belongs on it's asset page (whenever we build that feature) or on the crates.io/repo/webpage linked to imo

Reply:
This was just an option I was giving him, related comment: https://github.com/bevyengine/bevy-website/pull/394/files#r924157060
He used **"Option A"** in the end, which is just "ellipsis at the fourth line": https://github.com/bevyengine/bevy-website/pull/394/commits/1a93da13e2cdbe0d8d8b2ac4949dba917e76f775